### DISCUSSION TITLE
Possible to never pull an image?

### DISCUSSION THREAD

**Discussion by michael-brade** [woodpecker-ci/woodpecker#2813] [Request a Feature]
Hi, is it possible to configure woodpecker so that the build steps never pull an image? If it isn't available locally, I want it to throw an error instead. But [here](https://woodpecker-ci.org/docs/usage/pipeline-syntax#image) it says `pull: false` just means it won't update a tag if a newer version is available. drone had `pull: never`.

**Comment by qwerty287** [comment]
That's not possible yet. (I moved this to feature requests)

**Reply by michael-brade** [reply]
:+1: thanks!
